packages/workers-shared/asset-worker/src/index.ts

```diff
@@ -42,8 +42,22 @@ export default class extends WorkerEntrypoint<Env> {
 		}
 	}
 
+	async unstable_canFetch(...[request]: Parameters<UnstableCanFetchMethod>) {
```

Comment:
why `unstable`?

Reply:
may move to another entrypoint so it is not exposed to end users